uber/configspec.ini

```diff
@@ -543,6 +543,7 @@ teardown = string(default="Teardown")
 [[__many__]]
 __many__ = string
 
+[donation_tiers]
```

Comment:
Right now this section looks like this:
```
[donation_tiers]
[[no_donation]]
desc = 'No thanks'
default_price = 0

[[ribbon]]
desc = 'Ribbon'
default_price = 5
```

One of the things this lost us is the ability to assign integer enums to certain tiers. I'm not sure why, but it throws exceptions when I try to do so.